py-polars/tests/unit/io/test_csv.py

```diff
@@ -1460,30 +1460,41 @@ def test_csv_9929() -> None:
 
 
 def test_csv_quote_styles() -> None:
+    dt = date(1999, 12, 31)
+    dtm = datetime(2077, 7, 5, 3, 1, 0)
     df = pl.DataFrame(
         {
             "float": [1.0, 2.0, None],
             "string": ["a", "a,bc", '"hello'],
             "int": [1, 2, 3],
             "bool": [True, False, None],
+            "date": [dt, None, dt],
+            "datetime": [None, dtm, dtm],
```

Comment:
Maybe add a test case for Time datatype as well

Reply:
Why not indeed... done  ;)